src/sqs/push.ts

```diff
@@ -40,65 +37,3 @@ export const pushQueueMessageHandler : MessageHandler<PushQueueMessagePayload> =
 	enhanceOctokit(github);
 	await processPush(github, payload, wrapLogger(context.log));
 }
-
```

Comment:
Extracted into a common file https://github.com/atlassian/github-for-jira/pull/945/files#diff-94ace22f9f271ab4c8d87f9aa3778841f6401700260d4a25ad6524f55f89476cR1